pom.xml

```
@@ -108,12 +108,17 @@
     			<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-javadoc-plugin</artifactId>
-					<version>2.9</version>
+					<version>3.5.0</version>
 					<configuration>
 						<quiet>true</quiet>
-						<source>8</source>
+						<source>11</source>
```

```suggestion
						<source>8</source>
```

There's actually a project still using Java 8